src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
+  class="html"
+  lang="en"
```

Comment:
The `class` attribute on the `html` tag is not necessary unless it's being used for a specific purpose in CSS or JavaScript. If it's not used, it's better to remove it to keep the markup clean.